course/templates/course/course_base.html

```diff
@@ -79,26 +79,38 @@ <h4>{% trans "Calendar" %}</h4>
             {% language user_language %}
             <div lang={{user_language}}>
                 <div class="alert alert-default">
-                    {% trans "You are browsing this course in different language than set in your preferences." %}
+                    {% trans "You are browsing this course in a different language than is set in your preferences." %}
                     {% if user_course_data %}
                         <div class="row">
                             <div class="col-lg-4">
                                 <form action="{{ url_without_language }}">
-                                    <button class="btn btn-default btn-block" type="submit">{% blocktrans with lang=user_language|language_name_translated %}browse the course in {{ lang }}{% endblocktrans %}</button>
+                                    <button type="submit" class="aplus-button--secondary aplus-button--md btn-block">
```

Comment:
Is the `btn-block` required? If so, then the feature must be copied over to `aplus-button-*` system.

Reply:
Without `btn-block` it doesn't look very good:
![buttons-grey](https://user-images.githubusercontent.com/36920208/90523371-77780900-e175-11ea-8cbe-f0240013ff8b.png)

Reply:
Did you try without the columns? Just two buttons next to one another. 

I didn't implement the `*-block` because it wasn't being used much and generally left-aligned buttons in forms seem to follow the route the eye takes along the left-aligned field labels. 

But, of course we can implement one if we need it.

Reply:
Without columns: 
![buttons-grey2](https://user-images.githubusercontent.com/36920208/90537678-47853180-e186-11ea-945f-f4343545f95c.png)
Do you know an easy way to get the buttons next to each other? Now they are just one above the other.

Reply:
Yes! I'll send you an example in the morning.

Reply:
Would button groups be a suitable idea for these two buttons? (see https://github.com/apluslms/a-plus-design-system/issues/1#issuecomment-675594108 )

Reply:
In this particular case, I think it makes sense to have separate buttons. They're distinct actions with different effects.

Additionally, the way this PR is implemented now would be a problem, as each button is wrapped inside a `<form>`.

Reply:
@veeraalt how does it look now?